packages/thirdweb/src/wallets/in-app/core/authentication/types.ts

```diff
@@ -43,8 +45,8 @@ export type SingleStepAuthArgsType =
     }
   | {
       strategy: "siwe";
-      walletId: InjectedSupportedWalletIds;
-      chainId: number;
+      wallet: Wallet<InjectedSupportedWalletIds>;
```

Comment:
Does it only work with injected wallets?

Reply:
Hmm, I guess theoretically you could use any. I'm not sure why you would but I guess you can

Reply:
remember the wallet adapter use case